### PR TITLE
feat: update top pair spam strategy

### PR DESCRIPTION
**Pull request by ChefJerry**
### Summary
📊🆕🚀

Improved the logic of selecting the top 10 pairs to show on the overview page of the `Info` app. Changed the `poolsData` variable in `apps/web/src/views/Info/Overview/index.tsx` to use a `reduce` method instead of a `filter` method.

> _`poolsData` changed_
> _Using `reduce` method now_
> _Overview more fair_

### Walkthrough
* Improve the logic of selecting the top 10 pairs to show on the overview page by using a `reduce` method instead of a `filter` method ([link](https://github.com/pancakeswap/pancake-frontend/pull/8001/files?diff=unified&w=0#diff-e8968dd1774ec65b03552c785f82fa9eb9059e7f38af517de51a4bbae9d78924L63-R73))

### CODE REVIEW

**Comment by chefjackson** [comment]
kindly fix ts errors

**Comment by ChefJerry** [comment]
@chefjackson ts error resolved

File: apps/web/src/views/Info/components/InfoTables/TransactionsTable.tsx [hunk 1]
```diff
@@ -102,10 +102,11 @@ const DataRow: React.FC<React.PropsWithChildren<{ transaction: Transaction }>> =
   const abs1 = Math.abs(transaction.amountToken1)
   const chainName = useChainNameByQuery()
   const { domainName } = useDomainNameForAddress(transaction.sender)
+  const [address0, address1] = [transaction.token0Address, transaction.token1Address].map(isAddress)
```
Comment (chefjackson) [inline apps/web/src/views/Info/components/InfoTables/TransactionsTable.tsx:105]: `isAddress` is updated to `safeGetAddress` in https://github.com/pancakeswap/pancake-frontend/pull/7984

**Review by chefjackson — CHANGES_REQUESTED** [review]
kindly resolve the conflict

**Review by chefjackson — APPROVED** [review]

**Pull request merged**